### PR TITLE
Fix OpenSSL python script on Windows

### PR DESCRIPTION
**Pull request by OHerasym**
Fix generating certificates on Windows platform

### CODE REVIEW

**Comment by OHerasym** [comment]
Please review: @anosach-luxoft , @dev-gh

**Comment by dev-gh** [comment]
Reviewed.

File: tools/Utils/generate_test_certificates.py [hunk 1]
```diff
@@ -54,6 +55,12 @@ def gen_cert(out_cert_file, key_file, ca_cert_file, ca_key_file, days, answer):
     openssl("x509 -hash -req -in", request_file, "-CA", ca_cert_file, "-CAkey", ca_key_file, \
         "-CAcreateserial -out", out_cert_file, "-days", days)
 
+def convertPath(path):
+      sep = os.path.sep
```
Comment (anosach-luxoft) [inline tools/Utils/generate_test_certificates.py]: @OHerasym indents

Comment (OHerasym) [inline tools/Utils/generate_test_certificates.py]: @anosach-luxoft , Fixed

**Pull request merged**